### ISSUE TITLE
Stupid Repo

### ISSUE THREAD

**Issue by amanuel2** [uniforlyff/codingbuddies#1]
Such a dum repo..

**Comment by jdc20181** [comment]
lol

**Comment by uniforlyff** [comment]
You guys back lol this repo is abandoned. How's the ng-forum?

**Comment by jdc20181** [comment]
Hey Sens! ng-forum isn't great tbh aman is planning on finishing it but
wants to spend his time working on his other projects. How's things?

**Comment by amanuel2** [comment]
Isnt great? Are you fcking kidding me? Anways, yes im working on other big sellers... And ofcourse sens bieng sens, he always abandons projects.

**Comment by uniforlyff** [comment]
Haha

**Comment by uniforlyff** [comment]
I didnt abandon it, the others didnt know how to code

**Comment by uniforlyff** [comment]
I'll take a look at you guyses project tmrw and see it for myself

**Comment by jdc20181** [comment]
Aman it's the truth. Because you said it your self you was bout done with
it. Because you "don't like website programming" . So no I'm not fkin
kidding you. It's alive but slowly crippling.
Sens...How's things? Are you working on anything else? Aren't you in
college or skemthing?

**Comment by uniforlyff** [comment]
Its summer :D

**Comment by uniforlyff** [comment]
Yeah I think aman was going to game dev and which I am a game designer

**Comment by uniforlyff** [comment]
stuck between game design or web dev

**Comment by jdc20181** [comment]
xD Sens have fun! Enjoy summer don't do anything stupid lol

**Comment by uniforlyff** [comment]
Haha true some house burnt down

**Comment by uniforlyff** [comment]
Why do u think the site is slowly crippling?

**Comment by uniforlyff** [comment]
Who coded that register/login I like it

**Comment by jdc20181** [comment]
Oh lord. Ur in California tho so it is probally not uncommon..isn't there
some big fire over there somewhere...I outta look it up tomorrow lol I'm
going to bed I am actually replying from my email and on my phone. Haven't
been on computer today at all I don't get on much anymore. Like you said
it's summer I go to the Libary go walking or bike riding. I need to read
more but just get to busy...

**Comment by amanuel2** [comment]
Lol ok.. Yes idc aboutwebsite programming.. its to easy..

**Comment by amanuel2** [comment]
thats me... I coded almost everything. And the site is not updated...

**Comment by uniforlyff** [comment]
I see yeah I read from now and then so who coded the login/register page?

**Comment by amanuel2** [comment]
I have had tons of changes.... go here to see updated : https://preview.c9users.io/amanuel2/ng-fourm/index.html

**Comment by jdc20181** [comment]
It was between aman and shim. Aman uses a lot of examples I've noticed. But
it isn't bad. It could be better it isn't mobile friendly, we are using
gulp too now (I think I got that right lmao)
We have badges now well close to it me and aman was coding them the other
day k didn't do much lmao he wouldn't let me do front end so I had to sit
there and have him guide me through.
 I don't do backend stuff much mainly CSS and HTML (bootstrap jQuery mobile
JQuery etc perfer to leave our libraries tho)

**Comment by jdc20181** [comment]
At least what he was doing with badges I dot do.

**Comment by uniforlyff** [comment]
It crash my browser. Couldn't even get to pressed the white button

**Comment by jdc20181** [comment]
Oh wow Sens. Hmmm thanks for the info. What browser you using and OS?

**Comment by uniforlyff** [comment]
Android and firefox

**Comment by jdc20181** [comment]
Ok I'll see what I can do tomorrow. I will test it. See what happens. I
have a android tablet don't use it much but I'll dig it out. I need to work
on a older perfectly fine not even used a year computer the fan isn't good
in it but I don't think they sell the right size fan without ordering on
Amazon so I'm gonna hook it up to it from external :) eventually putting
Linux on it right now it has Windows 7 but I'll take care of it lol or dual
boot

**Comment by uniforlyff** [comment]
I see okay dude nice talking to ya, have to go later :)

**Comment by amanuel2** [comment]
Back i was studying Calculus.. Um, yea. Right now im majorly focusing on Mobile Friendliness but a good friend of mine, told me about media quieries.. So its kinda easy. RIght now though im trying to work on making the new topic responsive :/ . No idea how to make it though, ill do it somehow.

**Comment by uniforlyff** [comment]
Media queries will solve your 'mobile friendliness' issue

**Comment by jdc20181** [comment]
Probally so, I am thinking we could do something else but o need to read up
on material a little phones bout to die :/
